usr/src/lib/libzpool/common/kernel.c

```diff
@@ -346,7 +346,10 @@ cv_timedwait_hires(kcondvar_t *cv, kmutex_t *mp, hrtime_t tim, hrtime_t res,
 	ASSERT(flag == 0);
 
 top:
-	delta = tim - gethrtime();
+	delta = tim;
+	if (flag & CALLOUT_FLAG_ABSOLUTE)
```

Comment:
In DEBUG, this will never get hit, because the ASSERT() on line 346 will trigger if (flag & (ANYTHING_NON_ZERO)).